emails/templates/emails/wrapped_email.html

```diff
@@ -69,6 +69,6 @@
         </td>
       </tr>
     </table>
-    {{ original_html|safe }}
+    {{ original_html|linebreaks|safe }}
```

Comment:
Should we apply the `linebreaks` filter here, or should we apply it to `text_content` in the view?

If we apply this here, how does it affect HTML emails? Does it convert all the linebreaks in their markup into more `<p>` and `<br>` tags?

Reply:
Good catch I'll look into both!